core-plugins/pom.xml

```diff
@@ -216,6 +222,7 @@
               co.cask.hydrator.plugin.validator.*;
               org.apache.avro.mapreduce;
               parquet.avro.*;
+              ch.ethz.ssh2.*;
```

Comment:
Forget honestly. It was back when building a CDAP app in a testcase wasn't recognizing the package.